user/migrate/open-source-repository-migration.md

```diff
@@ -185,3 +185,49 @@ If you have any further questions, comments or need help on our Beta migration p
+
+
+#### Q. What will happen to travis-ci.org after December 31st, 2020?
+
+A. Travis-ci.org will be switched to a read-only platform, allowing you to see your jobs build history from all repositories previously connected to travis-ci.org.
+
+
+#### Q. Why are some of my queued jobs taking longer than usual to build?
+
+A. At peak usage times, you may see your build times are longer than they previously have been - we need to make sure all users have equal access to the .org platform until the end of the year as we move our infrastructure across to .com. You may want to consider migrating across to .com sooner rather than later, or consider scheduling builds at a quieter time of the day if remaining on .org for a while longer.
+
+
+#### Q. Why are some of my queued jobs being cancelled?
+
+A. This might happen as we move infrastructure over from .org to .com - if we see a job stuck in a queue for more than 16 hours, we will review this on a case by case basis and possibly cancel the job to ensure resources are not being reduced across the platform. You may want to consider migrating across to .com sooner rather than later, or consider building at a quieter time of the day if remaining on .org for a while longer.
+
+
+#### Q. Will there be lower concurrency for free accounts on travis-ci.org?
+
+A. As part of the shift of infrastructure from .org to .com and needing to make sure all users have equal access to resources, free and open-source .org accounts will have concurrency reduced from 5 to 4 concurrent jobs. Concurrent jobs have not changed on .com, so please consider migrating your repositories as soon as possible if this is an issue.
+
+
+#### Q. Will Travis-ci.org become unreliable?
+
+A. As part of the shift of infrastructure from .org to .com we’re making a number of changes to the travis-ci.org infrastructure to ensure the service will remain as reliable and available to you as it always has been until the 
+
+
+#### Q. Will Travis CI be getting rid of free users?
+
+A. Travis CI will continue to offer a free tier for public or open-source repositories on travis-ci.com and will not be affected by the migration.
+
+
+#### Q. Why is travis-ci.com asking for write access to my repositories?
+
+A. We’re aware that when migrating your GitHub repositories to travis-ci.com you will be prompted to give Travis CI write access to your repositories - this is due to us currently using GitHub OAuth for user authentication and the message is due to the way the OAuth scopes are shaped. The Travis CI platform actually uses the GitHub App for actual repository-level access - it does not require write access to all of your repositories and you can configure that.
+
+We’re working hard to resolve this and use just the GitHub App for both 
+user authentication and repository-level access, but until then we will be clarifying the situation in our documentation and user interface.
```

Comment:
We’re working hard to resolve this and to just use the GitHub App for both;
user authentication and repository-level access, but ....